icu4c/source/test/cintltst/reapits.c

```diff
@@ -55,8 +55,9 @@ log_err("Test Failure at file %s:%d - ASSERT(%s) failed.\n", __FILE__, __LINE__,
     status = U_ZERO_ERROR; \
     re = uregex_openC(pattern, flags, NULL, &status);  \
     TEST_ASSERT_SUCCESS(status);   \
-    srcString = (UChar *)malloc((strlen(testString)+2)*sizeof(UChar)); \
-    u_uastrncpy(srcString, testString,  strlen(testString)+1); \
+    int32_t testStringLen = (int32_t)strlen(testString); \
+    srcString = (UChar *)malloc( (testStringLen + 2) * sizeof(UChar) ); \
```

Comment:
Are the extra spaces intentional?

Reply:
I thought it might make it a bit clearer as to what malloc was allocating, but if not I can revert the changes for spacing.

Reply:
Per offline discussion, the spacing change isn't blocking as this is just test code.